portal-bridge/src/types/state/utils.rs

```diff
@@ -0,0 +1,24 @@
+use alloy_primitives::U256;
+
+pub fn u256_to_lower_u64(u256: U256) -> u64 {
```

Comment:
Actually, now that I understand better how this is used, I would say it should panic if number is bigger than u64.

In that case, you can just use `x.to::<u64>()` ([link](https://docs.rs/alloy-primitives/latest/alloy_primitives/aliases/type.U256.html#method.to)), and you probably don't need this function at all.